source/shared/net/desktop-wallet/multisig-simple-transfer.rst

```diff
@@ -32,8 +32,6 @@ Make a transfer proposal
 
 #.  Select **Send GTU**.
 
-#.  Select the relevant proposer. You’ll see a list of all the identities that you’re the custodian of. When you select an identity, the **Transactions Details** in the left pane are updated with the identity information and the current transaction expiry time. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
-
 #. Select the relevant sender account from the list of available accounts, and then select **Continue**.
```

Comment:
```suggestion
#. Select the relevant sender account from the list of available accounts, and then select **Continue**. If only one multi-signature account is available, this will be chosen automatically.
```